docs/markdown/Build-options.md

```diff
@@ -29,6 +30,8 @@ name.
 
 These options are accessed in Meson code with the `get_option` function.
 
+The array type is new in 0.43.0
```

Comment:
I think this should be 0.44.0 now, right?

Reply:
yes it should